packages/css-reset/README.md

```diff
@@ -1,3 +1,7 @@
-# TDS Core: CSS Reset
+```css noeditor
+@import '~@tds/core-css-reset/index';`
```

Comment:
This could also be imported into a .jsx file so webpack can pick it. We want people to use the dist/ one in either case. I think we prefer the jsx one so that the CSS loader can just take it and load it into the bundle. No need to pipe this file through Sass as we have already compiled it to CSS.

```
// in jsx
import '@tds/core-reset/dist/index.css'

// in Sass
import "~@tds/core-reset/dist/index"
```